pipelines/full-test-hpo.yml

```diff
@@ -1,38 +1,71 @@
 trigger: none
-pr: none
+
 # schedules:
 # - cron: 0 16 * * *
 #   branches:
 #     include: [ master ]
 
-jobs:
-- job: linux
-  pool: nni-it
-  timeoutInMinutes: 60
+resources:
+  repositories:
+    - repository: azpfilter
+      type: github
+      name: microsoft/azure-pipeline-filter
+      endpoint: github-filter-connection
+
+variables:
+  filter.modified.globs: 'examples/tuners/**,nni/algorithms/hpo/**'
+  filter.prbody.heading: '#### Test Options'
+  filter.prbody.optionIndex: 0
+
+stages:
+- stage: filter
+  jobs:
+  - job: check
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+    - checkout: azpfilter
+      persistCredentials: true
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '16.x'
+    - script: npm ci
+      displayName: NPM Install
+    - script: node src/main.mjs
+      name: execution
+      displayName: Execution
+
+- stage: test
+  # dependencies.$(StageName).outputs['$(JobName).$(TaskName).$(VariableName)']
```

Comment:
What is this?

Reply:
we can get output in one stage by this pattern
here we use:
`dependencies.filter.outputs['check.execution.skipsubsequent']`